src/disco/css/Addon.scss

```diff
@@ -6,7 +6,7 @@ $addon-padding: 20px;
 .addon {
   align-items: center;
   background: #fff;
-  box-shadow: 0 1px 2px 0 rgba(0,0,0,0.1);
+  box-shadow: 0 1px 2px 0 rgba(0,0,0,0.2);
```

Comment:
We're planning to introduce linting of css at some point, just haven't got to it yet. At that time it would make sense to formalize formatting for this. I agree that spaces after commas help readability though so I'll update all of these.